eng/common/pipelines/templates/jobs/archetype-spec-gen-sdk.yml

```diff
@@ -168,6 +168,34 @@ jobs:
         artifactName: $(sdkArtifactName)
         targetPath: "$(System.DefaultWorkingDirectory)/generatedSdkArtifacts"
 
+    - task: PowerShell@2
```

Comment:
I tend to move these label processing steps to the entry pipeline yml file in spec repo since this is specific to spec PR scenario. @weshaggard , what do you think?

Reply:
I'm open to that idea. We should talk about how we would enable that labeling from the github workflow.

Reply:
I will merge this PR for now as most of the steps will be in this pipeline template. It is a separated work while talking about labeling using github workflow.